### ISSUE TITLE
GitHub / Jekyll notebook error

### ISSUE THREAD

**Issue by sr320** [sr320/LabDocs#459]
Anyone else tried posting a notebook entry and getting an error like

```
The page build failed with the following error:

The value 'nil' was passed to a date-related filter that expects valid dates in `sitemap.xml` or one of its layouts. For more information, see https://help.github.com/articles/page-build-failed-date-is-not-a-valid-datetime/.

For information on troubleshooting Jekyll see:

  https://help.github.com/articles/troubleshooting-jekyll-builds

If you have any questions you can contact us by replying to this email.
```

via email?

I just made a copy of a markdown file that renders, changed 1 character and still get this error- has been happening for past two days

**Comment by kubu4** [comment]
Possibly related to your date formatting? Seems odd that it would become an issue now, but it's possible.

According to your `_layouts/post.html` file, your date should be in the following format (example):

#### January 31, 2017 ####

![sr320_github_io_post_html_at_master_ _sr320_sr320_github_io](https://cloud.githubusercontent.com/assets/4514104/22488938/b0ba8f66-e7c9-11e6-9600-25bc0f6d971a.png)





But, you're writing your posts with this format (example):

#### 2017-01-31 ####



![mozilla_firefox](https://cloud.githubusercontent.com/assets/4514104/22488978/e2c3b654-e7c9-11e6-8300-ec00eff80a28.png)



The incorrect date format could result in that `nil` value being passed along, which is triggering the error.


You could try changing your `_layouts/post.html` to have the format that matches your existing posts: `%Y-%m-%d`


Replace this line: `Written on {{ page.date | date: "%B %e, %Y" }}`

With this line: `Written on {{ page.date | date: "%Y-%m-%d" }}`

**Comment by sr320** [comment]
Will try- but all posts render in the format (except new posts in past day)
Including one you highlight

**Comment by sr320** [comment]
OK I think I fixed it and can guess as to the problem..

Last week I named a file 2017-01-30-41-DMRS.md

I figured the number in the name was an issue, so changed to Forty.

However I believe as the history of this still was a problem. And once there is a Jekyll snag, nothing else will post (I presume it runs through all files each time a new file is added).

Also I think you cannot place a `:` in the post title within the header of the post.
